lib/network/kmip/Client.ts

```diff
@@ -483,19 +493,20 @@ class Client {
 
     /**
      *
-     * @param {number} cryptoScheme - crypto scheme version number
-     * @param {string} masterKeyId - key to retrieve master key
-     * @param {buffer} plainTextDataKey - data key
-     * @param {object} logger - werelog logger object
-     * @param {function} cb - callback
-     * @returns {undefined}
+     * @param cryptoScheme - crypto scheme version number
+     * @param masterKeyId - key to retrieve master key
+     * @param plainTextDataKey - data key
+     * @param logger - werelog logger object
+     * @param cb - callback
      * @callback called with (err, cipheredDataKey: Buffer)
      */
-    cipherDataKey(cryptoScheme,
-        masterKeyId,
-        plainTextDataKey,
-        logger,
-        cb) {
+    cipherDataKey(
+        cryptoScheme: number,
+        masterKeyId: string,
+        plainTextDataKey: Buffer,
+        logger: werelogs.Logger,
+        cb: any,
```

Comment:
Maybe being more specific on the callbacks type is better? (maybe not, so it's just a suggestion)

```suggestion
        cb: Function,
```

Reply:
Usually we are writing callbacks as `(error: Error | null, data: YourDataType) => void` and this add type-checking. Some type-checking still needs to be added, so I prefer to let it as `any` to really indicates "This is not properly typed".